lightning/src/ln/channel.rs

```diff
@@ -1672,7 +1672,29 @@ impl<SP: Deref> Channel<SP> where
 				res
 			},
 			ChannelPhase::Funded(mut funded_channel) => {
-				let res = funded_channel.commitment_signed(msg, logger).map(|monitor_update_opt| (None, monitor_update_opt));
+				#[cfg(splicing)]
+				let session_received_commitment_signed = funded_channel
+					.interactive_tx_signing_session
+					.as_ref()
+					.map(|session| session.has_received_commitment_signed())
+					.unwrap_or(false);
```

Comment:
Could you help me understand the difference between the case where `session.has_received_commitment_signed()` is false and where we don't have an `interactive_tx_signing_session`?